docroot/themes/contrib/civictheme/civictheme_library/components/03-organisms/list/list.scss

```diff
@@ -67,7 +67,6 @@
   }
 
   &--with-background {
-    #{$root}__header,
```

Comment:
@joshua-salsadigital 

This is used for this case
<img width="1248" alt="News_and_events___CivicTheme_Source" src="https://user-images.githubusercontent.com/378794/202551886-78b0af1a-0047-4164-92da-bef4180c9261.png">

If this is removed - the title will 'stick' to the left side.

Can you please test with and without background on a page with and without sidebar (4 variants)